### ISSUE TITLE
Drush path cannot be written to

### ISSUE THREAD

**Issue by ccharlton** [lando/lando#984]
Lando Version: Beta.45

This message did not appear yesterday after a `lando rebuild`. Today, when running `lando start` this text barfs on the screen:

```
Directory /var/www/.drush exists, but is not writable. Please check directory permissions.                                                                                                                                                           [error]
 PHP configuration      :
 PHP OS                 :  Linux
 Drush script           :  /tmp/drush.phar
 Drush version          :  8.1.16
 Drush temp directory   :  /tmp
 Drush configuration    :
 Drush alias files      :

Directory /var/www/.drush exists, but is not writable. Please check directory permissions.                                                                                                                                                           [error]
PHP Fatal error:  Uncaught exception 'Pantheon\Terminus\Exceptions\TerminusException' with message 'Could not save data to a file because the path /var/www/.terminus/cache/commands cannot be written to.' in /var/www/html/vendor/pantheon-systems/terminus/src/DataStore/FileStore.php:155
Stack trace:
#0 /var/www/html/vendor/pantheon-systems/terminus/src/DataStore/FileStore.php(118): Pantheon\Terminus\DataStore\FileStore->ensureDirectoryWritable()
#1 /var/www/html/vendor/pantheon-systems/terminus/src/DataStore/FileStore.php(51): Pantheon\Terminus\DataStore\FileStore->getFileName('Pantheon\\Termin...', true)
#2 /var/www/html/vendor/consolidation/annotated-command/src/Cache/CacheWrapper.php(47): Pantheon\Terminus\DataStore\FileStore->set('Pantheon\\Termin...', Array)
#3 /var/www/html/vendor/consolidation/annotated-command/src/AnnotatedCommandFactory.php(175): Consolidation\AnnotatedCommand\Cache\CacheWrapper->set('Pantheon\\Termin...', Array)
#4 /var/www/html/vendor/consolidation/annotated-command/src/AnnotatedCommandFactory.php(159 in /var/www/html/vendor/pantheon-systems/terminus/src/DataStore/FileStore.php on line 155

Fatal error: Uncaught exception 'Pantheon\Terminus\Exceptions\TerminusException' with message 'Could not save data to a file because the path /var/www/.terminus/cache/commands cannot be written to.' in /var/www/html/vendor/pantheon-systems/terminus/src/DataStore/FileStore.php on line 155

Pantheon\Terminus\Exceptions\TerminusException: Could not save data to a file because the path /var/www/.terminus/cache/commands cannot be written to. in /var/www/html/vendor/pantheon-systems/terminus/src/DataStore/FileStore.php on line 155

Call Stack:
    0.0041     261936   1. {main}() /var/www/html/vendor/pantheon-systems/terminus/bin/terminus:0
    0.7253    7597688   2. Pantheon\Terminus\Terminus->run() /var/www/html/vendor/pantheon-systems/terminus/bin/terminus:44
    0.7254    7598216   3. Robo\Runner->run() /var/www/html/vendor/pantheon-systems/terminus/src/Terminus.php:103
    0.7254    7598536   4. Robo\Runner->registerCommandClasses() /var/www/html/vendor/consolidation/robo/src/Runner.php:190
    0.7255    7620408   5. Robo\Runner->registerCommandClass() /var/www/html/vendor/consolidation/robo/src/Runner.php:229
    0.7278    7702992   6. Consolidation\AnnotatedCommand\AnnotatedCommandFactory->createCommandsFromClass() /var/www/html/vendor/consolidation/robo/src/Runner.php:263
    0.7278    7703144   7. Consolidation\AnnotatedCommand\AnnotatedCommandFactory->getCommandInfoListFromClass() /var/www/html/vendor/consolidation/annotated-command/src/AnnotatedCommandFactory.php:148
    0.7416    8112536   8. Consolidation\AnnotatedCommand\AnnotatedCommandFactory->storeCommandInfoListInCache() /var/www/html/vendor/consolidation/annotated-command/src/AnnotatedCommandFactory.php:159
    0.7425    8144368   9. Consolidation\AnnotatedCommand\Cache\CacheWrapper->set() /var/www/html/vendor/consolidation/annotated-command/src/AnnotatedCommandFactory.php:175
    0.7425    8144424  10. Pantheon\Terminus\DataStore\FileStore->set() /var/www/html/vendor/consolidation/annotated-command/src/Cache/CacheWrapper.php:47
    0.7425    8144472  11. Pantheon\Terminus\DataStore\FileStore->getFileName() /var/www/html/vendor/pantheon-systems/terminus/src/DataStore/FileStore.php:51
    0.7425    8144808  12. Pantheon\Terminus\DataStore\FileStore->ensureDirectoryWritable() /var/www/html/vendor/pantheon-systems/terminus/src/DataStore/FileStore.php:118

error: Looks like one of your build steps failed...
warn: This **MAY** prevent your app from working
warn: Check for errors above, fix them, and try again
```

**Comment by ccharlton** [comment]
But, the site/app loads fine. FWIW.

**Comment by pirog** [comment]
@ccharlton fixed this as part of #975 
https://github.com/lando/lando/commits/master

If you `lando rebuild` the updated images should contain the fix. Otherwise there is an in-code fix which will be in the Friday release.
